src/woo_publications/publications/tests/test_publications_api.py

```diff
@@ -2090,6 +2079,212 @@ def test_updated_publication_schedules_index_task(
 
 
 class PublicationApiRequiredFieldsTestCase(TokenAuthMixin, APITestCase):
+    def test_create_concept_with_only_officiele_titel(self):
+        url = reverse("api:publication-list")
+        data = {
+            "publicatiestatus": PublicationStatusOptions.concept,
+            "officieleTitel": "title one",
+        }
+
+        response = self.client.post(url, data, headers=AUDIT_HEADERS)
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+    def test_update_concept_with_only_officiele_titel(self):
+        publication = PublicationFactory.create(
+            officiele_titel="title one",
+            publicatiestatus=PublicationStatusOptions.concept,
+        )
+        url = reverse(
+            "api:publication-detail",
+            kwargs={"uuid": str(publication.uuid)},
+        )
+        data = {
+            "publicatiestatus": PublicationStatusOptions.concept,
+            "officieleTitel": "update",
+        }
+
+        response = self.client.put(url, data, headers=AUDIT_HEADERS)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
200 OK, means we can check a body. Something like:
```
for field, value in response.json()["wherethepublicationlives"]:
    if field != "officiele_title":
        self.assertEqual(value, getattr(publication, decamelize(field))
```

I'm merging this to unblock ICatt, but since DRF straddles the edge of PUT semantics, it's good to check this. (in a separate PR)